source/rules/roles.txt

```diff
@@ -205,9 +205,11 @@ App Services assigns roles at different times depending on whether you're using
 :ref:`Device Sync (Flexible Mode) <sync>` or not.
 
 When using Device Sync, App Services assigns roles at the start of a Sync
-session for each collection to be synced; otherwise, App Services assigns roles
-on a per-document basis. A Sync session is the period of time between opening
-and closing a Sync connection.
+session for each collection to be synced. A Sync session is the period of time
+between opening and closing a Sync connection.
+
+When not using Device Sync, App Services assigns roles on a per-document,
```

Comment:
Stupid (?) question: if I'm not using Sync, I'm not using App Services at all -- just local data, no?